mlflow/recipes/steps/train.py

```diff
@@ -160,7 +165,10 @@ def _validate_and_apply_step_config(self):
                 error_code=INVALID_PARAMETER_VALUE,
             )
 
-        self.predict_proba = self.step_config.get("predict_proba", False)
+        self.predict_prefix = self.step_config.get("predict_prefix", "predicted_")
+        self.predict_scores_for_all_classes = self.step_config.get(
+            "predict_scores_for_all_classes", True
+        )
```

Comment:
Same with this conf - have stakeholders signed off? cc @jinzhang21

Reply:
There is a P1 requirement:
```
[P1] I can configure it to always predict the scores for both labels. So I can easily get the score of a certain label, e.g., the probability of click (positive) in CTR prediction.
Example: {“predicted_label”: “true”, “predicted_score”: 0.85, “predicted_score_true”: 0.85, “predicted_score_false”: 0.15}
```
So added this conf based on that. 

We don't need to document them right away since we have sensible defaults for it. :)

Reply:
Sounds good!